backend/__tests__/userGetBookings.test.js

```
@@ -0,0 +1,179 @@
+        token: mockToken
+      }
+    });
+    
+    res = httpMocks.createResponse();
+    
+    // Setup process.env
+    process.env.JWT_SECRET = 'test-secret';
+  });
+
+  afterEach(() => {
+    delete process.env.JWT_SECRET;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)